build.gradle

```diff
@@ -227,3 +232,15 @@ bootJar {
 wrapper {
     distributionType = Wrapper.DistributionType.ALL
 }
+
+task migratePostgresDatabase(type: org.flywaydb.gradle.task.FlywayMigrateTask) {
+  baselineOnMigrate = true
```

Comment:
baseline properties are not needed for this new DB.

Reply:
Fixed in https://github.com/hmcts/cpo-case-payment-orders-api/pull/16/commits/3f932e0a934d18ed4fb8e61d94ed18b6058a32f9